tailwind.config.js

```diff
@@ -15,6 +15,20 @@ module.exports = {
       backgroundImage: (theme) => ({
         favicon: "url( './favicon.png')",
       }),
+      colors: {
+        'orange-custom': '#FBA239',
```

Comment:
`custom` is a bad name. Please use Tailwind conventions on naming, i.e. `orange-500`, etc. Please consult designers. Dipali updated the naming in Figma.

Reply:
Designers need not only to update the names of colors but also codes because orange-500 is color: #ed8936 in tailwind orange-custome is color: #FBA239.  I took a call with Olesya she raised a question on a call with other designers on Friday, and she confirmed that we can go with current names.